aderyn_driver/src/foundry_compiler_helpers.rs

```diff
@@ -17,6 +18,10 @@ pub fn get_compiler_input(root: &Path) -> CompilerInput {
         .into_iter()
         .filter(|c| c.language == *"Solidity")
         .collect::<Vec<_>>();
+    if solidity_files.is_empty() {
```

Comment:
Test here too please

Reply:
This is to check for safe array index access in the next line ` let solidity_files = &solidity_files[0]; `

We must ensure vector is not empty if we want to access index 0 of the vector : )

Reply:
Idea - 💡 Let me write a test for this by having a directory with no solidity files at all whatsoever.